spec/jekyll_seo_tag_spec.rb

```diff
@@ -196,6 +196,60 @@
       end
     end
 
+    context "with image.path, image.height, and image.width" do
```

Comment:
Where might you suggest this goes? It didn't feel right placing it with the other "with image.path only" (Or around that) test because it looks like the tests are broken into `<meta>` and `json-ld` sections.